### PR TITLE
#5660 add classes for different version modications

### LINKED ISSUE #5660: class name "versionmodified" used by all versionchanged/added/deprecated
Subject: HTML output for change log (I want to CSS the headlines)

### Problem
The current implementation use the same class name for all three version roles. This means I cannot use CSS to have the color of the text reflect if its added (green), changed (yellow) or deprecated (red). The only difference between them is the text itself but for the reader this might become hard to read through when its all the same color (default black).

http://www.sphinx-doc.org/en/1.6/extdev/nodes.html#sphinx.addnodes.versionmodified

#### Expected results
Role | ClassName
------------ | -------------
versionadded | versionmodified.added
versionchanged | versionmodified.changed
deprecated | versionmodified.deprecated

### Reproducible project / your project
- https://espeasy.readthedocs.io/en/latest/

### Environment info
- Sphinx version: 1.8.1

### PR DESCRIPTION
**Pull request by TimKam**
Subject: Add classes for different version modifications

### Feature or Bugfix
- Feature

### Purpose
- To allow users to use different styles for different modification types.
### Detail
* Generic class: ``versionmodified``
* *Changed* class: ``changed``
* *Added* class: ``added``
* *Deprecated* class: ``deprecated``

### Relates
- Closes #5660

### CODE REVIEW

File: sphinx/domains/changeset.py [hunk 1]
```diff
@@ -27,13 +27,18 @@
     from sphinx.application import Sphinx  # NOQA
     from sphinx.environment import BuildEnvironment  # NOQA
 
-
```
Comment (tk0miya) [inline sphinx/domains/changeset.py]: Please keep this line as is :-)

**Review by tk0miya — CHANGES_REQUESTED** [review]
LGTM with nits.
In addition, you need to modify some testcases.
Please check errors on CI servers.

**Review by tk0miya — APPROVED** [review]
Looks good!
Go ahead. Add an entry to CHANGES file and merge this :-)

**Comment by TimKam** [comment]
Ok, when the build on ``master`` no longer fails, I will control merge and merge back.

**Comment by tk0miya** [comment]
Oh, sorry. I just fixed master branch.

**Pull request merged**